### PR TITLE
TensorFlow ClusterSpec example

### PR DESCRIPTION
**Pull request by roscisz**

### CODE REVIEW

File: examples/TF_CONFIG/README.md [hunk 1]
```diff
@@ -93,20 +95,20 @@ to specify batch size, we enter parameter name --batch_size, again select Static
...
 
 Note that the TF_CONFIG and CUDA_VISIBLE_DEVICES variables are configured automatically. Now, use
 the `ADD TASK` button to duplicate the processes and modify the required target hosts to create
 your training processes. For example, this screenshot shows the configuration for training on 4
 hosts: gl01, gl02, gl03, gl04:
 
-![multi_process](https://github.com/roscisz/TensorHive/tree/master/examples/TF_CONFIG/img/multi_process.png)
+![multi_process](https://raw.githubusercontent.com/roscisz/TensorHive/master/examples/TF_CONFIG/img/multi_process.png)
 
 After clicking the `CREATE ALL TASKS` button, the processes will be available in the process list for future actions.
```
Comment (mikolaj395) [inline examples/TF_CONFIG/README.md]:
```suggestion
After clicking the `CREATE ALL TASKS` button, the processes will be available on the process list for future actions.
```

File: examples/TensorFlow_ClusterSpec/README.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+processes, they could be handled by `ADD AS PARAMETER TO ALL TASKS`.
+
+Use `ADD TASK` to create as many copies of the defined process as required and
+select the appropriate hostname and resource (CPU/GPU_N) for the specified training process. The resultant
+command that will be executed by TensorHive on the selected node will be displayed above the process specification.
+Note that the cluster parameters and CUDA_VISIBLE_DEVICES variable are configured automatically:
+ 
+![ready](https://raw.githubusercontent.com/roscisz/TensorHive/master/examples/TensorFlow_ClusterSpec/img/ready.png)
+
+After clicking the `CREATE ALL TASKS` button, the processes will be available in the process list for future actions.
```
Comment (mikolaj395) [inline examples/TensorFlow_ClusterSpec/README.md]:
```suggestion
After clicking the `CREATE ALL TASKS` button, the processes will be available on the process list for future actions.
```

File: examples/TensorFlow_ClusterSpec/README.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+
+The task creator allows also to conveniently specify other command-line arguments. For example,
+should the `train_files`, `dev_files` and `test_files` parameters change throughout the training
+processes, they could be handled by `ADD AS PARAMETER TO ALL TASKS`.
+
+Use `ADD TASK` to create as many copies of the defined process as required and
+select the appropriate hostname and resource (CPU/GPU_N) for the specified training process. The resultant
+command that will be executed by TensorHive on the selected node will be displayed above the process specification.
+Note that the cluster parameters and CUDA_VISIBLE_DEVICES variable are configured automatically:
```
Comment (mikolaj395) [inline examples/TensorFlow_ClusterSpec/README.md]: Don't we have to change parameter server process `job_name` parameter value to `ps` manually?

File: examples/deepspeech/README.md [hunk 4]
```diff
@@ -4,5 +4,7 @@ The distributed DeepSpeech training application that had been used as a test
 application and requirement provider towards TensorHive is now in a
 [separate repository](https://github.com/roscisz/dnn_training_benchmarks/tree/master/TensorFlowV1_DeepSpeech_ldc93s1).
 
-See the [TensorHive examples directory](https://github.com/roscisz/TensorHive/tree/master/examples) for
-examples how TensorHive task execution module can be used for various training applications. 
\ No newline at end of file
+The application has been also used in the usage example of the TensorHive task execution
+module for the [TensorFlow ClusterSpec](https://github.com/roscisz/TensorHive/tree/master/examples/TensorFlow_ClusterSpec)
+scenario. See also the [TensorHive examples directory](https://github.com/roscisz/TensorHive/tree/master/examples) for other
+examples how TensorHive task execution module can be used for various training scenarios.
```
Comment (mikolaj395) [inline examples/deepspeech/README.md]: Sounds a bit awkward to me, maybe something like:
> The application has been also used in the TensorHive task execution
> module example for the [TensorFlow ClusterSpec](https://github.com/roscisz/TensorHive/tree/master/examples/TensorFlow_ClusterSpec)
> scenario. See also the [TensorHive examples directory](https://github.com/roscisz/TensorHive/tree/master/examples) for other
> examples how TensorHive task execution module can be used for various training scenarios.

Comment (roscisz) [reply examples/deepspeech/README.md]: fixed in b42594add4945f72a9a9a97d703699c179a3de39

File: examples/TensorFlow_ClusterSpec/README.md (hunk 3 above)
Comment (roscisz) [reply examples/TensorFlow_ClusterSpec/README.md]: indeed, added description in bae71bdd663852f8e34230adb395e6b12910ca02

**Pull request merged**